### ISSUE TITLE
Direct IPA

### ISSUE THREAD

**Issue by Benevolize** [SpotCompiled/SpotilifeC#19]
I don't have the beta version of AltStore, is there a direct IPA to install?

**Comment by yodaluca23** [comment]
Yes of course, go to the [release tab](https://github.com/SpotCompiled/SpotilifeC/releases).
